src/com/gmail/St3venAU/plugins/ArmorStandTools/Main.java

```diff
@@ -172,7 +172,7 @@ void setName(Player p, ArmorStand as) {
             p.sendMessage(ChatColor.RED + Config.noAirError);
             return;
         }
-        b.setType(Material.SIGN);
+        b.setType(Material.OAK_WALL_SIGN);
```

Comment:
Shouldn't this be a Sign post?

Reply:
SIGN_POST isn't a Material in 1.15 API.
Maybe the Material would be OAK_SIGN.
I'm not sure how that part is supposed to work.

However if the plugin is intended to be compiled in 1.13 just remove that part before merging.

Reply:
From what I understand is the method(s) used with this used for things like renaming the armor stand or adding a Player head.
In that case does AST "summon" a Sign which is "opened" for the player to then input the text.
And it was always a standing sign. I'm not sure if a wall sign would change this dramatically, but I would leave it as a standing sign for consistency.

Reply:
I checked and it works the same. However I just change it from OAK_WALL_SIGN to OAK_SIGN